images/pulp-minimal/stable/Containerfile.core

```diff
@@ -27,7 +27,17 @@ RUN pip3 install --upgrade \
 
 # Prevent pip-installed /usr/local/bin/pulp-content from getting run instead of
 # our /usr/bin/pulp-content script.
-RUN rm -f /usr/local/bin/pulp-content
+# RUN rm -r /usr/local/bin/pulp-content
```

Comment:
I know this is a hack, but we need it for older pulpcore.

Reply:
I'm checking it, but seems I'm unable to revert it. :sob:

Reply:
Don't confuse `-f` and `-r` as i did.